subliminal/providers/addic7ed.py

```diff
@@ -108,7 +82,7 @@ def get(self, url, params=None):
 
         """
         try:
-            r = self.session.get(self.server + url, params=params, timeout=10)
+            r = self.session.get(self.server + url, params=params, timeout=30)
```

Comment:
I think i had a failure one day and got lazy and changed the value;  i'll flip it back.